pyslackers_website/templates/slack/partials/member_history_chart.html

```diff
@@ -0,0 +1,55 @@
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.min.js"></script>
```

Comment:
We are now using `yarn` and `webpack` to build our frontend, so this should be made a dependency in the `package.json` with: `yarn add chart.js` (or whatever the library is named).
  

It's also kind of a bad practice to sprinkle `script` tags around html - generally it should either be at the beginning (and made async or deferred) or just before the body close.

Reply:
Yeah. I tried splitting this up, following the how the map was implemented, but I think it boiled down to not understanding how to use webpack (or how to construct the json, frankly). Tried to follow what was already there but nothing would work. I'll give it another shot tomorrow.

Reply:
Sounds good! I am on a branch trying to make the current example a (little) better - maybe this will help: https://github.com/pyslackers/website/blob/f1b7c70763be8b37e72585e5e9bdaff859519e58/pyslackers_website/templates/slack/index.html#L21

Basically: set the JSON in a `<script type="application/json" id="your_data_id">{{ thing | tojson }}</script>` and retrieve it in the JS.

To run the JS build see [Item 7 under here](https://github.com/pyslackers/website/blob/master/CONTRIBUTING.md#run-the-things)